pkg/convert/json_ngxtranslate.go

```diff
@@ -0,0 +1,72 @@
+package convert
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+var ErrWrongKey = errors.New("Wrong key type")
```

Comment:
1) In go, it's idiomatic to start an error message with a lowercase letter.
2) Declare package errors only if the package consumers require to check for specific errors. I'm not sure if this is applicable to the given package.

```suggestion
var ErrWrongKey = errors.New("wrong key type")
```